attest/tpm_windows.go

```diff
@@ -42,6 +42,10 @@ type windowsTPM struct {
 func (*windowsTPM) isTPMBase() {}
 
 func probeSystemTPMs() ([]probedTPM, error) {
+	// Initialize Tbs.dll here so that it's linked only when TPM support is required.
+	tbs = windows.MustLoadDLL("Tbs.dll")
```

Comment:
Can you surround this in a check to only load the DLL if it hasnt already been loaded? ie: `tbs == nil`

Reply:
sure, thanks!